.github/workflows/test-dsk.yml

```diff
@@ -0,0 +1,17 @@
+
+name: Test DSK image
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "dsk/0.0.100/Dockerfile"
+
+jobs:
+
+  build-to-test:
+    uses: StaPH-B/docker-builds/.github/workflows/build-to-test.yml@master
```

Comment:
If you changed this to `./.github/workflows/build-to-test.yml`, does it trigger a workflow?

Reply:
Indeed it does, I wonder why though. I'll take another read through the workflow docs. Anyways, should we update the template file accordingly here? https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/test-template.yml#L19

Also I did find in the docs that the workflow approval is only triggered the first time someone contributes, after that workflows run automatically.